template/rwa-onboarding/Goerli-DssSpellCollateral.sol

```diff
@@ -21,38 +19,50 @@ pragma solidity 0.6.12;
 // pragma experimental ABIEncoderV2;
 
 import "dss-exec-lib/DssExecLib.sol";
-import "dss-interfaces/dapp/DSTokenAbstract.sol";
-import "dss-interfaces/dss/ChainlogAbstract.sol";
 import "dss-interfaces/dss/GemJoinAbstract.sol";
 import "dss-interfaces/dss/IlkRegistryAbstract.sol";
-import "dss-interfaces/dss/JugAbstract.sol";
-import "dss-interfaces/dss/SpotAbstract.sol";
-import "dss-interfaces/dss/VatAbstract.sol";
 import "dss-interfaces/ERC/GemAbstract.sol";
 
-import "./test/addresses_goerli.sol";
-
 interface RwaLiquidationLike {
     function ilks(bytes32) external returns (string memory, address, uint48, uint48);
     function init(bytes32, uint256, string calldata, uint48) external;
 }
 
 interface RwaUrnLike {
+    function vat() external view returns(address);
+    function jug() external view returns(address);
+    function gemJoin() external view returns(address);
+    function daiJoin() external view returns(address);
+    function outputConduit() external view returns(address);
     function hope(address) external;
-    function lock(uint256) external;
+}
+
+interface RwaJarLike {
+    function chainlog() external view returns(address);
+    function dai() external view returns(address);
+    function daiJoin() external view returns(address);
 }
 
 interface RwaOutputConduitLike {
+    function dai() external view returns(address);
+    function gem() external view returns(address);
+    function psm() external view returns(address);
+    function file(bytes32 what, address data) external;
     function hope(address) external;
     function mate(address) external;
+    function kiss(address) external;
 }
 
 interface RwaInputConduitLike {
+    function dai() external view returns(address);
+    function gem() external view returns(address);
+    function psm() external view returns(address);
+    function to() external view returns(address);
     function mate(address usr) external;
+    function file(bytes32 what, address data) external;
```

Comment:
[This PR](https://github.com/makerdao/dss-interfaces/pull/75) has just been merged, so it would be good to update `dss-interfaces` dependency and reference the contracts from [this directory](https://github.com/makerdao/dss-interfaces/tree/master/src/dss/mip21) instead of re-declaring them.